django_cloud_tasks/signals.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from django.core.exceptions import ValidationError
+from django.db.models.signals import post_save, pre_save
+from django.dispatch import receiver
+
+from django_cloud_tasks import models, tasks
+
+
+@receiver(post_save, sender=models.Routine)
+def enqueue_routine_task_after_creation(sender, instance, created, **kwargs):
+    if created:
+        tasks.RoutineTask(routine_id=instance.pk)
```

Comment:
It does not trigger anything like this: it should be `tasks.RoutineTask().delay(routine_id=instance.pk)`